packages/wallet/contract/src/walletFactory.js

```diff
@@ -34,12 +44,32 @@ export const start = async (zcf, privateArgs) => {
   assert(namesByAddress, 'missing namesByAddress');
   assert(agoricNames, 'missing agoricNames');
   const zoe = zcf.getZoeService();
-  const { storageNode } = privateArgs;
+  const { storageNode, bridgeManager } = privateArgs;
 
   /** @type {MapStore<string, import('./smartWallet').SmartWallet>} */
   const walletsByAddress = makeScalarBigMapStore('walletsByAddress');
   const provider = makeAtomicProvider(walletsByAddress);
 
+  const handleWalletAction = Far('walletActionHandler', {
+    /**
+     *
+     * @param {string} srcID
+     * @param {WalletAction} obj
+     */
+    fromBridge: async (srcID, obj) => {
+      console.log('walletFactory.fromBridge:', srcID, obj);
+      assert(obj, 'missing wallet action');
+      assert.typeof(obj, 'object');
+      assert.typeof(obj.owner, 'string');
+      const wallet = walletsByAddress.get(obj.owner); // or throw
+      console.log('walletFactory:', { wallet });
+      return E(wallet).performAction(obj); // TODO: add performAction to lib-wallet.js
```

Comment:
I'm not sure what value `fromBridge` has when `performAction` doesn't exist but I'll trust you do :)

Reply:
This didn't get past `yarn lint` so made the whole like a TODO comment.